Description: fix svnversion.h dependency to permit parallel building
 With CMake 3.9.1 and parallel building enabled, the incorrect specification
 of the output filename when creating svnversion.h can lead to a race condition
 such that the package fails to build.
Author: Stuart Prescott <stuart@debian.org>
Bug: https://sourceforge.net/p/rosegarden/bugs/1552/
Bug-Debian: https://bugs.debian.org/871213
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -810,7 +810,7 @@
 add_custom_target(svnheader ALL DEPENDS svn_header) # svn_header never gets created, it's just a unique string
 
 # creates svnheader.h using cmake script
-add_custom_command(OUTPUT svn_header ${CMAKE_CURRENT_BINARY_DIR}/svnheader.h
+add_custom_command(OUTPUT svn_header ${CMAKE_CURRENT_BINARY_DIR}/svnversion.h
     COMMAND ${CMAKE_COMMAND} -DSOURCE_DIR=${CMAKE_CURRENT_SOURCE_DIR}
                          -P ${CMAKE_CURRENT_SOURCE_DIR}/buildkey.cmake)
 
